lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,7 +1,7 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
   <div class="sidebar_body">
-    <ul id="archives">
+    <ul id="archives" type="circle">
```

Comment:
Why did you add `type="circle"` here and not anywhere else? Do the bullets that work use `type="circle"`?